vignettes/a03_rsyncrosim_vignette_pipelines.Rmd

```diff
@@ -105,51 +105,51 @@ When creating a new modeling workflow from scratch, we need to create objects of
 
 For more information on these scopes, see the [Introduction to `rsyncrosim`](https://syncrosim.github.io/rsyncrosim/rsyncrosim_vignette_basic.html) vignette.
 
-### Set up Library, Project, and Scenario
+### Set up library, project, and scenario
 
 ```{r create Library, warning = FALSE}
-# Create a new Library
+# Create a new library
 myLibrary <- ssimLibrary(name = "helloworldLibrary.ssim",
```

Comment:
Did you test running through this vignette? The `package` argument is now `packages` so this bit of code should have failed.